plugin/src/main/java/io/jenkins/plugins/remotingkafka/KafkaComputerLauncher.java

```diff
@@ -137,6 +146,45 @@ public void onClosed(Channel channel, IOException cause) {
         }
     }
 
+    private void launchKubernetesPod(KafkaCloudComputer computer) {
+        try {
+            KafkaCloudSlave slave = computer.getNode();
+            if (slave == null) {
+                throw new IllegalStateException("Node has been removed, cannot launch " + computer.getName());
+            }
+            KubernetesClient client = slave.getCloud().connect();
```

Comment:
Is this autocloseable?

Reply:
I use try-with-resource in the `connect` method ([link](https://github.com/jenkinsci/remoting-kafka-plugin/blob/61f4a5dbffab98cc43bc933c6b675d624dc88324/plugin/src/main/java/io/jenkins/plugins/remotingkafka/KafkaKubernetesCloud.java#L69)). Do I still need to close connection at the end of this method?